packages/styles/scss/themes/expressive/components/_code-snippet.scss

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/layout/scss/convert';
+
+//-----------------------------
+// Code snippet (expressive)
+//-----------------------------
+
+/// Code snippet styles (expressive)
+/// @access private
+/// @group code-snippet-expressive
+@mixin code-snippet-expressive {
+  // Single Line Snippet
+  .#{$prefix}--snippet--single {
+    height: $carbon--spacing-09;
+  }
+  .#{$prefix}--snippet--single .#{$prefix}--snippet-container {
+    height: $carbon--spacing-09;
+    padding: 0;
+  }
+  .#{$prefix}--snippet-button {
+    height: 3rem;
+    width: 3rem;
+  }
+  //Single Line Snippet Copy Button
+  .#{$prefix}--snippet__icon {
+    height: rem(20px);
+    width: rem(20px);
+  }
+  // Multi Line Snippet Copy Button
+  .#{$prefix}--snippet--multi .#{$prefix}--snippet-button {
+    height: $carbon--spacing-08;
+    width: $carbon--spacing-08;
+  }
+  .#{$prefix}--btn--ghost {
+    border-width: 1px;
```

Comment:
why are we changing the border width here?

Reply:
@annawen1 In the ticket description saying the button height would be 40px but it has 44px, when I looked it has 1px border is adding from four sides it is becoming 44px so I reduced the border-width to 1px.

Reply:
mmm we probably don't want to do that because it will affect the focus state of the buttons

ie: left is border-width of 3px and middle button is border-width of 1px
<img width="297" alt="Screen Shot 2020-01-17 at 11 04 15 AM" src="https://user-images.githubusercontent.com/54281166/72626721-2b33c480-3919-11ea-87a0-89def336261c.png">

You might want to check the `<Button />` component (small variation) styles and see why it's not 40px

Reply:
Okay